### PR TITLE
Don't update all gems, allow to install specific versions

### PR DESCRIPTION
**Pull request by njam**
Closes https://github.com/cargomedia/osx-setup/issues/117

### CODE REVIEW

**Comment by njam** [comment]
@ppp0 please review

File: resource/osx/install/_default/scripts/ruby.sh [hunk 1]
```diff
@@ -1,19 +1,27 @@
 GEM='/usr/local/opt/ruby/bin/gem'
 
 function gemInstall {
-	if ! (${GEM} list --no-versions | grep -q $1); then
-		${GEM} install $1
-	fi
+  if [ -n "${2}" ]; then
+    if ! (${GEM} list --installed --version "${2}" "${1}"); then
```
Comment (ppp0) [inline resource/osx/install/_default/scripts/ruby.sh]: Here you reference "${2}" but in line 4 you check if it's NULL? ( `-n` is null/not initialized ?)

**Comment by ppp0** [comment]
may be a logic error (see above)? Rest looks good

File: resource/osx/install/_default/scripts/ruby.sh (hunk 1 above)
Comment (njam) [inline resource/osx/install/_default/scripts/ruby.sh]:
```
     -n string     True if the length of string is nonzero.
```

Comment (njam) [inline resource/osx/install/_default/scripts/ruby.sh]: agree, `test` is confusing…

**Pull request merged**